prow/jobs/jinja/presubmits/service_tests.jinja2

```diff
@@ -78,13 +80,13 @@
     spec:
       serviceAccountName: pre-submit-service-account
       containers:
-      - image: public.ecr.aws/bitnami/aws-cli:2.2.43
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.7
```

Comment:
Note to self: this needs to use the image context

Reply:
what is the image context?